### DISCUSSION TITLE
Custom Wallpapers

### DISCUSSION THREAD

**Discussion by Oskar-Idland** [prasanthrangan/hyprdots#751] [Q&A] [answered]
Hello, I tried reading the restoring scripts, but I am pretty new to bash scripting and could not find the answer. 

My question is, if I add my own wallpapers in for example `~/.config/swww/Graphite-Mono` as i only want the wallpaper to be available there, will it still be there after running `install.sh -r` ? 

I was also wondering if there was some way to permanently remove some of the wallpapers?

**Comment by kRHYME7** [comment, marked as answer]
In your cloned repository ```  ~/Hyprdots/Scripts/restore_cfg.lst ``` this is the file that controls your restoration to default. So you can edit this before you run ``` install.sh -r ```. So 1st column write Y if you want to copy the defaults to your .configs and in the 2nd column write Y If you want to back up.
(BTW I made a CLI https://github.com/kRHYME7/Hyprdots-ctl Try to check that out it is just a CLI for Hyprdots.) 

```
 cat ~/Hyprdots/Scripts/restore_cfg.lst
N|Y|${HOME}|.zshrc .p10k.zsh|zsh oh-my-zsh-git zsh-theme-powerlevel10k pokemon-colorscripts-git
Y|Y|${HOME}/.config/Code/User|settings.json|visual-studio-code-bin
Y|Y|${HOME}/.config|dunst|dunst
Y|Y|${HOME}/.config/hypr|animations.conf hyprland.conf theme.ctl windowrules.conf scripts themes wallbash|hyprland
N|Y|${HOME}/.config/hypr|keybindings.conf monitors.conf userprefs.conf|hyprland
N|Y|${HOME}/.config/hypr|nvidia.conf|hyprland nvidia-utils
Y|Y|${HOME}/.config|kitty|kitty
Y|Y|${HOME}/.config|Kvantum|kvantum
Y|Y|${HOME}/.config|MangoHud|mangohud
Y|Y|${HOME}/.config|neofetch|neofetch
Y|Y|${HOME}/.config|qt5ct|qt5ct
Y|Y|${HOME}/.config|rofi|rofi
Y|Y|${HOME}/.config|swaylock|swaylock-effects
Y|N|${HOME}/.config|swww|swww
Y|Y|${HOME}/.config|waybar|waybar
Y|Y|${HOME}/.config|wlogout|wlogout
Y|Y|${HOME}/.config|nwg-look gtk-3.0 xsettingsd|nwg-look
Y|Y|${HOME}/.icons|default|nwg-look
Y|Y|${HOME}|.gtkrc-2.0|nwg-look
Y|Y|${HOME}/.config|dolphinrc kdeglobals|dolphin
Y|Y|${HOME}/.local/share|dolphin|dolphin
Y|Y|${HOME}/.local/share/kxmlgui5|dolphin|dolphin
```



Yes,you can permanently delete the wallpapers by going to each of the folder like the folder you mentioned ``` ~/.config/swww/* ``` and deleting its equivalent cache at the (might not be needed) ``` ~/.cache/hyprdots/theme-name ```

**Reply by Oskar-Idland** [reply]
Thank you!
